public/app/features/profile/ChangePasswordForm.tsx

```diff
@@ -34,29 +35,40 @@ export const ChangePasswordForm: FC<Props> = ({ user, onChangePassword, isSaving
           return (
             <>
               <Field label="Old password" invalid={!!errors.oldPassword} error={errors?.oldPassword?.message}>
-                <Input type="password" {...register('oldPassword', { required: 'Old password is required' })} />
+                {/* <Input type="password" {...register('oldPassword', { required: 'Old password is required' })} /> */}
+                <PasswordField
+                  id="current-password"
+                  autoComplete="current-password"
+                  register={register('oldPassword', { required: 'Old password is required' }) as any}
+                />
               </Field>
 
               <Field label="New password" invalid={!!errors.newPassword} error={errors?.newPassword?.message}>
-                <Input
-                  type="password"
-                  {...register('newPassword', {
-                    required: 'New password is required',
-                    validate: {
-                      confirm: (v) => v === getValues().confirmNew || 'Passwords must match',
-                      old: (v) => v !== getValues().oldPassword || `New password can't be the same as the old one.`,
-                    },
-                  })}
+                <PasswordField
+                  id="new-password"
+                  autoComplete="new-password"
+                  register={
+                    register('newPassword', {
+                      required: 'New password is required',
+                      validate: {
+                        confirm: (v) => v === getValues().confirmNew || 'Passwords must match',
+                        old: (v) => v !== getValues().oldPassword || `New password can't be the same as the old one.`,
+                      },
+                    }) as any
+                  }
                 />
               </Field>
 
               <Field label="Confirm password" invalid={!!errors.confirmNew} error={errors?.confirmNew?.message}>
-                <Input
-                  type="password"
-                  {...register('confirmNew', {
-                    required: 'New password confirmation is required',
-                    validate: (v) => v === getValues().newPassword || 'Passwords must match',
-                  })}
+                <PasswordField
+                  id="new-password"
```

Comment:
Should this be `confirm-password`? Otherwise we'll have two elements with the same ids.

Reply:
addressed